김효중/handle_css.js

```diff
@@ -0,0 +1,35 @@
+let major = document.querySelector('.Magor');
+major.addEventListener('click', () => {
+  let major_target = document.querySelector('.detail_major');
+  major_target.classList.toggle('change_to_flex');
+});
```

Comment:
비슷한 아래 경우 모두 해당되는 이야긴데, `click` 이벤트시 마다 `querySelector`를 통해 DOM 요소에 접근하고 있어요.

이벤트 핸들러 밖 스코프에서 작성한다면 불필요하게 DOM 요소를 탐색하는 것을 방지할 수 있어요

```suggestion
const major_target = document.querySelector('.detail_major');

major.addEventListener('click', () => {
  major_target.classList.toggle('change_to_flex');
});
```

> 물론 상황에 따라 다르겠지만요!